plugins/inputs/supervisor/supervisor.go

```diff
@@ -0,0 +1,183 @@
+package supervisor
+
+import (
+	"net/url"
+
+	"fmt"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/filter"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/kolo/xmlrpc"
```

Comment:
We usually group import into "built-in/standard libraries", "3rd party libraries" and "telegraf packages" and sort them alphabetically within the groups
```suggestion
	"fmt"
	"net/url"

	"github.com/kolo/xmlrpc"

	"github.com/influxdata/telegraf"
	"github.com/influxdata/telegraf/filter"
	"github.com/influxdata/telegraf/plugins/inputs"
```

Reply:
Included in upcoming commit